File: pyg-parent/pyg-sellergoods-interface/src/main/java/com/pyg/service/BrandService.java
package com.pyg.service;

import entity.PageResult;
import com.pyg.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {

    List<TbBrand> findAll();

    PageResult getPageBrand(Integer page, Integer rows);

    void add(TbBrand brand);

    void update(TbBrand brand);

    TbBrand findOne(Long id);

    void delete(long[] ids);

    PageResult findPage(TbBrand brand,Integer page,Integer size);

    List<Map> selectOptionList();
}

File: pyg-parent/pyg-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService", function ($http) {

    this.findPage = function (page,size) {
        return $http.get("../brand/findPage.do?page=" + page + "&size=" + size);
    }

    this.findOne = function (id) {
        return $http.get('../brand/findOne.do?id=' + id);
    }

    //增加
    this.add=function(entity){
        return  $http.post('../brand/add.do',entity );
    }
    //修改
    this.update=function(entity){
        return  $http.post('../brand/update.do',entity );
    }

    this.dele=function (ids) {
        return $http.get('../brand/delete.do?ids=' + $scope.selectIds);
    }

    //搜索
    this.search=function(page,rows,searchEntity){
        return $http.post('../brand/search.do?page='+page+"&rows="+rows, searchEntity);
    }

    this.selectOptionList=function () {
        return $http.get('../brand/selectOptionList.do');
    }

});